### ISSUE TITLE
enabling and applying the plugin breaks the functionality of auxiliary buttons (CTL-480)

### ISSUE THREAD

**Issue by wojciechsacewicz** [shmkle/TemporalResampler#5]
Hi!
Enabling and applying the plugin breaks the functionality of auxiliary buttons (CTL-480).

Pls fix

**Comment by shmkle** [comment]
Hello, are you sure there no other filters involved and it is just temporal resampler? I was not able to recreate this behavior with my s620. however your issue report made me notice a bug on my other otd filter, mablet mapping, the fix is now up in submission but it was giving the same behavior you described.

**Comment by wojciechsacewicz** [comment]
Yeah, I've tried reinstalling the whole program and installing ONLY
temporal resampler and the issue happened. After that I only use radial
follow smoothing and i didnt install it when trying out the temporal
resampler issue.

**Comment by wojciechsacewicz** [comment]
Also I am sure this is temporal resampler since when enabling, applying it
happens and when disabling, applying it fixes the aux buttons
